### ISSUE TITLE
Suggest widget details have low contrast

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode#111242]
<img width="600" src="https://user-images.githubusercontent.com/35271042/100124676-d1e9cb00-2e30-11eb-95d2-287c56cee8c9.png">

Unfortunately the low opacity here is making the text fail for color contrast. This happens almost any time we tweak opacity because our default foreground just passes the cc ratio and going lower makes it fail:

<img width="400" src="https://user-images.githubusercontent.com/35271042/100124567-abc42b00-2e30-11eb-9e34-06f65c2aed75.png">

If we bump the opacity to `0.60` it passes but doesn't look as great either:

<img width="600" src="https://user-images.githubusercontent.com/35271042/100124956-1b3a1a80-2e31-11eb-97b5-bc077c296b4c.png">

**Comment by jrieken** [comment]
> If we bump the opacity to 0.60 it passes but doesn't look as great either:

Do you have other ideas? Otherwise I'd go with .6

**Comment by misolori** [comment]
@jrieken only other thing would be to reduce the amount of text shown in the detail (if it can be tweaked):

![image](https://user-images.githubusercontent.com/35271042/100129949-fe084a80-2e36-11eb-83f7-0490e7347652.png)

**Comment by jrieken** [comment]
Not sure what you mean by that?

**Comment by misolori** [comment]
Trying to minimize the amount of visual noise from the details text, I was suggesting you could limit the amount of characters shown.

**Comment by jrieken** [comment]
That's not really an option - the whole point here is to show more information so that you can make a better selection from the suggestions

**Comment by jrieken** [comment]
@misolori Can you check this again. Poking at this with devtools shows good (✅) CC

<img width="910" alt="Screen Shot 2021-10-04 at 11 01 59" src="https://user-images.githubusercontent.com/1794099/135823538-7d1d130b-9d7e-425e-9e5d-ab5e647e5a3d.png">

**Comment by misolori** [comment]
yea this looks like it improved 👍
